submissions/maryOkosun/php/unitConverter.php

```diff
@@ -0,0 +1,15 @@
+<?php>
+    function unitConverter($value){
+        $pound = $value * 2.20462;
+        $yard = $value * 1.09361;
+        $fahrenheit = $value * (9/2)+32;
```

Comment:
I think it is `9/5`

Reply:
Thanks @adaugochi 
I have made the necessary changes. Please review. Thanks